src/romaine/parser/__init__.py

```diff
@@ -0,0 +1,34 @@
+from romaine.parser import (
+    simple,
+    multiline,
+    step,
+    section,
+    feature,
+    )
+from romaine.parser.exceptions import (  # noqa
+    UnclosedPythonishString,
+    MalformedTableError,
+    )
+
+
+class Parser(object):
+    """
+        Gherkin parser for romaine core.
```

Comment:
Well, if we put in the docs for the project which documentation style we prefer (there's an issue open for that anyway) then I'm happy as long as we're consistent. Even if we're consistently ugly. *smiles*